src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -31,27 +36,73 @@
 
     @FindBy(id = "login_button")
     private ExtendedWebElement loginBtn;
+    @FindBy(id = "login_button")
+    private ExtendedWebElement signUpButton;
 
     public LoginPage(WebDriver driver) {
         super(driver);
     }
 
+
     @Override
     public void typeName(String name) {
         nameInputField.type(name);
         hideKeyboard();
     }
 
+
+    public String getTypeName() {
+        return nameInputField.getText();
+    }
+
+    public String getTypePassword() {
+        return passwordInputField.getText();
+    }
+
+
+    @Override
+    public boolean verifyFieldsArePresented() {
+        return nameInputField.isElementPresent() &&
+                passwordInputField.isElementPresent() &&
+                maleRadioBtn.isElementPresent() &&
+                femaleRadioBtn.isElementPresent() &&
+                privacyPolicyCheckbox.isElementPresent();
+    }
+
+
     @Override
     public void typePassword(String password) {
         passwordInputField.type(password);
     }
 
+
+    @Override
+    public boolean isMaleRadioBtnUnchecked() {
+        return maleRadioBtn.isChecked();
+    }
+
+    @Override
+    public boolean isFemaleRadioBtnChecked() {
+        return femaleRadioBtn.isChecked();
+    }
+
+
     @Override
     public void selectMaleSex() {
-        maleRadioBtn.click();
+        femaleRadioBtn.click();
     }
 
+    @Override
+    public boolean isPrivacyPolicyCheckboxUnchecked() {
+        return privacyPolicyCheckbox.isChecked();
+    }
+
+    @Override
+    public boolean isPrivacyPolicyCheckboxChecked() {
+        return privacyPolicyCheckbox.isChecked();
+    }
```

Comment:
its two method which return the same results 
remove isPrivacyPolicyCheckboxUnchecked